CHANGELOG.md

```diff
@@ -0,0 +1,128 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
+
+## [Unreleased]
+
+### Added
+
+- Coveralls support (#149)
+- Reusable testing workflow (integration testing with latest Tarantool)
+- Simple CI based on GitHub actions (#114)
+- Support UUID type in msgpack (#90)
+- Go modules support (#91)
+- queue-utube handling (#85)
+
+### Fixed
+
+- Fix queue tests (#107)
+- Make test case consistent with comments (#105)
+
+### Changed
+
+- Handle everything with `go test` (#115)
+- Use plain package instead of module for UUID submodule (#134)
+- Reset buffer if its average use size smaller than quater of capacity (#95)
+
+## [1.5] - 2019-12-29
+
+First release.
+
+### Fixed
+
+- Fix `Upsert()` for `ConnectionMulti` infinite recursive call
+- Fix index out of range panic on `dial()` to short address
+- Fix cast in `defaultLogger.Report` (#49)
+- Fix race condition on extremely small request timeouts (#43)
+- Fix notify for `Connected` transition
+- Fix reconnection logic and add `Opts.SkipSchema()`
+- Fix future sending
+- Fix panic on disconnect + timeout
+- Fix block on msgpack error
+- Fix ratelimit
+- Fix timeouts
+- Fix mistake
```

Comment:
It is also really unclear: every "Fixed" list item could be described as "Fix mistake". If commit is also unclear, I would've prefer to not list it at all since this item contribution is none.

Reply:
Updated some changelog entries to make them more clear for users.

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index a4070f8..7385dd4 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,7 +11,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 ### Added
 
 - Coveralls support (#149)
-- Reusable testing workflow (integration testing with latest Tarantool)
+- Reusable testing workflow (integration testing with latest Tarantool) (#123)
 - Simple CI based on GitHub actions (#114)
 - Support UUID type in msgpack (#90)
 - Go modules support (#91)
@@ -34,25 +34,24 @@ First release.
 
 ### Fixed
 
-- Fix `Upsert()` for `ConnectionMulti` infinite recursive call
+- Fix infinite recursive call of `Upsert` method for `ConnectionMulti`
 - Fix index out of range panic on `dial()` to short address
 - Fix cast in `defaultLogger.Report` (#49)
 - Fix race condition on extremely small request timeouts (#43)
 - Fix notify for `Connected` transition
-- Fix reconnection logic and add `Opts.SkipSchema()`
+- Fix reconnection logic and add `Opts.SkipSchema` method
 - Fix future sending
 - Fix panic on disconnect + timeout
 - Fix block on msgpack error
 - Fix ratelimit
-- Fix timeouts
-- Fix mistake
-- Fix race condition
+- Fix `timeouts` method for `Connection`
+- Fix possible race condition on extremely small request timeouts
 - Fix race condition on future channel creation
 - Fix block on forever closed connection
-- Fix race condition in Connection
+- Fix race condition in `Connection`
 - Fix extra map fields
 - Fix response header parsing
-- Fix reconnect
+- Fix reconnect logic in `Connection`
 
 ### Changed
 
@@ -78,20 +77,20 @@ First release.
 
 ### Added
 
-- Add `Future.WaitChan()` and `Future.Err()` (#86)
+- Add methods `Future.WaitChan` and `Future.Err` (#86)
 - Get node list from nodes (#81)
-- Add method deleteConnectionFromPool()
+- Add method `deleteConnectionFromPool`
 - Add multiconnections support
-- Add `Addr()` for the connection (#64)
-- Add `Delete()` method for the queue
+- Add `Addr` method for the connection (#64)
+- Add `Delete` method for the queue
 - Implemented typed taking from queue (#55)
-- Add `OverrideSchema()` method for the connection
+- Add `OverrideSchema` method for the connection
 - Add default case to default logger
 - Add license (BSD-2 clause as for Tarantool)
-- Add `GetTyped()` method for the connection (#40)
-- Add `ConfiguredTimeout()` for the connection, change queue interface
-- Add example for queue
-- Add `GetQueue()` method for the queue
+- Add `GetTyped` method for the connection (#40)
+- Add `ConfiguredTimeout` method for the connection, change queue interface
+- Add an example for queue
+- Add `GetQueue` method for the queue
 - Add queue support
 - Add support of Unix socket address
 - Add check for prefix "tcp:"
@@ -99,12 +98,12 @@ First release.
 - Add note about magic way to pack tuples
 - Add notification about connection state change
 - Add workaround for tarantool/tarantool#2060 (#32)
-- Add `ConnectedNow()` method for the connection
+- Add `ConnectedNow` method for the connection
 - Add IO deadline and use `net.Conn.Set(Read|Write)Deadline`
 - Add a couple of benchmarks
 - Add timeout on connection attempt
 - Add `RLimitAction` option
-- Add `Call17()` for the connection to make a call compatible with Tarantool 1.7
+- Add `Call17` method for the connection to make a call compatible with Tarantool 1.7
 - Add `ClientParallelMassive` benchmark
 - Add `runtime.Gosched` for decreasing `writer.flush` count
 - Add `Eval`, `EvalTyped`, `SelectTyped`, `InsertTyped`, `ReplaceTyped`, `DeleteRequest`, `UpdateTyped`, `UpsertTyped` methods
@@ -112,17 +111,13 @@ First release.
 - Add `CallTyped` method
 - Add possibility to pass `Space` and `Index` objects into `Select` etc.
 - Add custom MsgPack pack/unpack functions
-- Add support Tarantool 1.6.8 schema format
-- Add support Tarantool 1.6.5 schema format
+- Add support of Tarantool 1.6.8 schema format
+- Add support of Tarantool 1.6.5 schema format
 - Add schema loading
 - Add `LocalAddr` and `RemoteAddr` methods for the connection
-- Add Upsert method for the connection
-- Add `Eval()` and `EvalAsync()` methods for the connection
+- Add `Upsert` method for the connection
+- Add `Eval` and `EvalAsync` methods for the connection
 - Add Tarantool error codes
 - Add auth support
 - Add auth during reconnect
 - Add auth request
-
-## [alpha] - 2015-01-16
-
-Alpha release. See added changes in Git.
```

Alpha release has been removed from changelog as it is not a public release.